### ISSUE TITLE
[Bug] Initial firmware installation intermittently fails [Windows 11 x64]

### ISSUE THREAD

**Issue by EdHerdman** [RPCS3/rpcs3#12690]
### Quick summary

In issue #12621 I reported RPCS3 would reliably crash to desktop after installing the firmware and clicking through the "firmware and LLE modules installed" message, before compiling the modules. Since that build, I've tried installing the firmware in multiple versions and occasionally RPCS3 will crash in the same way on the first attempt, but succeed on the second attempt by simply re-launching the emulator and overwriting the firmware. The behavior is improved, but still unreliable.

rpcs3-v0.0.24-14173-3581c5b0_win64 based on #12686 still has this behavior - I unzipped the files into a fresh directory, immediately tried to install the firmware through drag-and-drop, and it crashed immediately after the initial installation and showing the "firmware and LLE modules installed" message on the first run.

UPDATE: rpcs3-v0.0.24-14183-a604394b_win64 from #12700 succeeded on installing firmware on the first attempt. Hopefully this sticks...

### Attach a log file

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/9618040/RPCS3.log.gz)

### System configuration

Windows 11 Pro 22H2 (build 22621.521).
Ryzen 7 3700X

**Comment by Augusto7743** [comment]
Please test 0.0.24-14123.
Since 0.0.24-14127 happening wrong huge memory allocation crashing Linux.

**Comment by EdHerdman** [comment]
In #12621 I found the commit which originally seemed to introduce the bug, and the last build without the bug; both are newer than 0.0.24-14123. I'm also only testing Windows builds for now.

**Comment by Asinin3** [comment]
> I might have aborted firmware install on this

Sounds like you are making entirely new RPCS3 folders each time. You don't need to. Just put the executable inside RPCS3 folder. I keep around 300 exe files in my folder of different builds for fast regression tracking.

**Comment by EdHerdman** [comment]
Yes, you're right, but tracking whether RPCS3 firmware install succeeds was the entire purpose of reporting this issue.

**Comment by EdHerdman** [comment]
I can confirm this is still happening occasionally: rpcs3-v0.0.24-14337-5210df68_win64

Not a deal-breaker, but something odd is still happening behind the scenes causing an unwelcome CTD.

**Comment by digant73** [comment]
@EdHerdman is that old bug still present?

**Comment by EdHerdman** [comment]
Possibly a bad RAM issue from years back. This hasn't happened in a long time!
